packages/cli/tests/images/build.js

```diff
@@ -9,40 +9,40 @@ exports.default = {
 
 	'ssr-build/ssr-bundle.css': 1281,
 	'ssr-build/ssr-bundle.css.map': 2070,
-	'ssr-build/ssr-bundle.js': 10863,
-	'ssr-build/ssr-bundle.js.map': 49128,
+	'ssr-build/ssr-bundle.js': 14400,
+	'ssr-build/ssr-bundle.js.map': 63913,
 	'ssr-build/asset-manifest.json': 76,
 
-	'bundle.79d07.js': 21560,
-	'bundle.79d07.js.map': 85822,
-	'bundle.79d07.legacy.js': 22549,
-	'bundle.79d07.legacy.js.map': 106841,
+	'bundle.528e9.js': 26608,
+	'bundle.528e9.js.map': 107005,
+	'bundle.528e9.legacy.js': 28182,
+	'bundle.528e9.legacy.js.map': 133666,
```

Comment:
Note to self: this is a rather sizeable jump. Split chunks are no longer paying the cost of code splitting (as seen by `route-profile`'s change from 3198b -> 743b), but this is still rather extreme. ~5kb difference in main bundle for ~2.5kb in chunks doesn't seem right.